docs/pages/guides/spend-permissions/api-reference/spendpermissionmanager.mdx

```diff
@@ -0,0 +1,99 @@
+| `token`     | `address`| Token address (ERC-7528 native token address or ERC-20 contract).           |
+| `allowance` | `uint160`| Maximum allowed value to spend within each `period`.                        |
+| `period`    | `uint48` | Time duration for resetting used `allowance` on a recurring basis (seconds).|
+| `start`     | `uint48` | Timestamp this spend permission is valid after (unix seconds).              |
+| `end`       | `uint48` | Timestamp this spend permission is valid until (unix seconds).              |
+| `salt`      | `uint256`| An arbitrary salt to differentiate unique spend permissions with otherwise identical data. |
+| `extraData` | `bytes`  | Arbitrary data to include in the signature.                                 |
+
+#### `PeriodSpend`
+Describes the cumulative spend for the current active period.
+
+| Field       | Type     | Description                                                                 |
+|-------------|----------|-----------------------------------------------------------------------------|
+| `start`     | `uint48` | Start time of the period (unix seconds).                                    |
+| `end`       | `uint48` | End time of the period (unix seconds).                                      |
+| `spend`     | `uint160`| Accumulated spend amount for period.                                        |
+
+---
+
+### Contract functions
+#### `approve`
+
+Approve a spend permission via a direct call from the `account`. Only callable by the `account` specified in the spend permission.
+
+```solidity
+function approve(SpendPermission calldata spendPermission) external;
+```
+
+---
+
+#### `approveWithSignature`
+
+Approve a spend permission via a signature from the `account` owner. Compatible with [ERC-6492](https://eips.ethereum.org/EIPS/eip-6492) signatures for automatic account creation if needed.
+
+```solidity
+function approveWithSignature(SpendPermission calldata spendPermission, bytes calldata signature) external;
+```
+
+---
+
+#### `spend`
+
+Spend tokens using a spend permission, transferring them from the `account` to the `spender`. Only callable by the `spender` specified in the permission.
+
+```solidity
+function spend(SpendPermission memory spendPermission, uint160 value) external requireSender(spendPermission.spender);
+```
+
+---
+
+#### `revoke`
+
+Revoke a spend permission, permanently disabling its use. Only callable by the `account` owner specified in the spend permission.
+
+```solidity
+function revoke(SpendPermission calldata spendPermission) external requireSender(spendPermission.account);
+```
+
+---
+
+#### `getHash`
+
+Generate a hash of a `SpendPermission` struct for signing, in accordance with [EIP-712](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md).
+
+```solidity
+function getHash(SpendPermission memory spendPermission) public view returns (bytes32);
+```
+
+---
+
+#### `isApproved`
+
+Check if a spend permission is approved and not revoked.
```

Comment:
can we be explicit here about expired? i.e. will it return true if an expired permission is not revoked?